rust/gitxetcore/src/git_integration/mod.rs

```diff
@@ -1,7 +1,22 @@
+pub mod command_entry;
 pub mod git_commits;
 pub mod git_file_tools;
-pub mod git_integration_plumb;
-pub mod git_notes_wrapper;
-pub mod git_repo;
+pub mod git_merkledb;
+mod git_notes_wrapper;
+mod git_process_wrapping;
+mod git_repo_paths;
+mod git_repo_plumbing;
+pub mod git_repo_salt;
+mod git_xet_repo;
+
 pub mod git_url;
-pub mod git_wrap;
+pub mod git_user_config;
+pub mod git_version_checks;
+
+pub use crate::git_integration::git_xet_repo::git_repo_test_tools; // HERE
```

Comment:
Can we drop the `crate::git_integration::` in the beginning. Also, I'm thinking if we can drop all `pub` from mod [...] above and re-export their internal functions so imports in other source files can be concise.

Reply:
I tried doing just that, but there are a bunch of places (like the version checks) that are only called in one place, and the names beneath them aren't very precise.  So I went back to doing it like this for the lesser used modules.  We can still use git_integration::*; then e.g. git_user_config::verify_users to call them.